### ISSUE TITLE
训练一种新的语言该怎么设置如下内容？

### ISSUE THREAD

**Issue by Wentao795** [FunAudioLLM/CosyVoice#314]
感谢大佬的工作，如果我要训练法语，下面的language 需要修改吗？ 期待回复
get_tokenizer: !name:whisper.tokenizer.get_tokenizer
    multilingual: True
    num_languages: 100
    language: 'en'
    task: 'transcribe'

**Comment by aluminumbox** [comment]
no, whisper has french language label, check whisper.tokenizer.get_tokenizer detail and add <|fr|>

**Comment by Wentao795** [comment]
thanks~
